tests/sanity/ignore-2.17.txt

```diff
@@ -0,0 +1,37 @@
+plugins/module_utils/client/discovery.py import-3.7!skip
```

Comment:
Should we also remove the sanity ignore files for older ansible versions?

Reply:
Prepped a PR https://github.com/ansible-collections/kubernetes.core/pull/667